mock-core-configs/etc/mock/alma-8-x86_64.cfg

```diff
@@ -0,0 +1,24 @@
+include('/etc/mock/templates/alma-8.tpl')
+include('/etc/mock/templates/epel-8.tpl.rpmnew')
+
+config_opts['root'] = 'alma-8-x86_64'
```

Comment:
This can be put into the template file and structured to change correctly per-architecture.